src/python/pants_release/start_release.py

```diff
@@ -16,24 +16,48 @@
 
 import requests
 from packaging.version import Version
-from pants_release.git import git, git_fetch
+from pants_release.common import CONTRIBUTORS_PATH, VERSION_PATH, sorted_contributors
+from pants_release.git import git, git_fetch, github_pr_create
+
+from pants.util.strutil import softwrap
 
 logger = logging.getLogger(__name__)
 
+BASE_PR_BODY = softwrap(
+    """
+    This PR is release preparation. Please read over the release notes and make adjustments for
+    clarity for users, such as:
+
+    - shuffle miscategorised items to a better category (including completely uncategorised items)
+
+    - fix or improve poor titles, like "fix bug", so they are more accurate or descriptive (check to
```

Comment:
The PR title is the commit message now.

Reply:
Usually yeah, but someone might update the PR title after merge (I know I forget to update them sometimes and go back to fix it up).